spec/models/placements/mentor_spec.rb

```diff
@@ -44,4 +44,11 @@
       expect(described_class.all).not_to include(claims_mentor)
     end
   end
+
+  describe "#full_name" do
```

Comment:
This test should be unnecessary given https://github.com/DFE-Digital/itt-mentor-services/blob/32fc251c93508bed6a4706859c0982ae3e90a166/spec/models/mentor_spec.rb#L63